examples/todo/client/src/components/App.css

```diff
@@ -0,0 +1,125 @@
+/*
+ * Base structure
+ */
+
+/* Move down content because we have a fixed navbar that is 50px tall */
+body {
+  padding-top: 50px;
+}
+
+
+/*
```

Comment:
Why variadic count of newlines before and after CSS classes and comments in this file?